crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -102,9 +102,20 @@ pub struct BloomSettings {
     /// configured in a non-energy-conserving way,
     /// otherwise set to [`BloomCompositeMode::EnergyConserving`].
     pub composite_mode: BloomCompositeMode,
+
+    // Maximum size of each dimension for the largest mipchain texture used in downscaling/upscaling.
```

Comment:
This need to be triple backslashes to show up in the docs.

In case you didn't know: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html